__tests__/app.test.js

```diff
@@ -618,4 +618,46 @@ describe('API', () => {
                 })
         });
     });
+
+    describe('DELETE /API/COMMENTS/:COMMENT_ID', () => {
+        test('Status 204 - api point exists and returns', () => {
+            return request(app)
+                .delete("/api/comments/1")
+                .expect(204);
+        });
+        test('Status 204 - response should be undefined', () => {
+            return request(app)
+                .delete("/api/comments/1")
+                .then(({ body }) => {
+                    expect(body).toEqual({});
+                })
+        });
+        test('Status 204 - comment with id 1 should be removed from DB', () => {
+            return request(app)
+                .delete("/api/comments/1")
+                .expect(204)
+                .then(() => {
+                    return db.query('SELECT * FROM comments WHERE comment_id = 1;')
```

Comment:
While this works, doing a request to the endpoint would be a better way of checking this.